decidim-core/app/presenters/decidim/metric_charts_presenter.rb

```diff
@@ -19,19 +19,27 @@ def highlighted
     def not_highlighted
       not_highlighted_metrics = Decidim.metrics_registry.not_highlighted
       safe_join(
-        not_highlighted_metrics.map do |metric|
-          render_metrics_data(metric.metric_name, klass: "small")
+        not_highlighted_metrics.in_groups_of(2, [:empty]).map do |metrics_group|
+          content_tag :div, class: "column medium-4" do
+            safe_join(
+              metrics_group.map do |metric|
+                render_metrics_data(metric.metric_name, klass: "column medium-6", graph_klass: "small")
+              end
+            )
+          end
         end
       )
     end
 
     private
 
     def render_metrics_data(metric_name, opts = {})
-      content_tag :div, "", id: "#{metric_name}_chart", class: "areachart metric-chart #{opts[:klass]}",
-                            data: { chart: "areachart", metric: metric_name,
-                                    info: { title: I18n.t("decidim.metrics.#{metric_name}.title"),
-                                            object: I18n.t("decidim.metrics.#{metric_name}.object") } }
+      content_tag :div, class: opts[:klass].presence || "column medium-6" do
+        content_tag :div, "", id: "#{metric_name}_chart", class: "areachart metric-chart #{opts[:graph_klass]}",
```

Comment:
Can you put each argument in a single line, please? That'd improve readability.

Reply:
Ok